libs/api/domains/license-service/src/lib/dto/GenericLicenseError.dto.ts

```diff
@@ -0,0 +1,25 @@
+import { Field, Int, ObjectType } from '@nestjs/graphql'
+import { GenericLicenseType } from '../licenceService.type'
+import { GenericLicenseProvider } from './GenericLicenseProvider.dto'
+import { GenericLicenseFetch } from './GenericLicenseFetch.dto'
+
+@ObjectType('GenericLicenseError')
+export class GenericLicenseError {
+  @Field(() => GenericLicenseType)
+  type!: GenericLicenseType
+
+  @Field(() => GenericLicenseFetch, { description: 'Info about license fetch' })
+  fetch!: GenericLicenseFetch
+
+  @Field(() => GenericLicenseProvider, { nullable: true })
+  provider?: GenericLicenseProvider
+
+  @Field(() => Int, { nullable: true })
+  errorCode?: number
```

Comment:
~nit: Since this is already a error class cant you just remove the error prefix from errorCode and errorMessage?

Reply:
true